src/main/js/services/PipelineStore.js

```diff
@@ -43,12 +45,10 @@ function _copy<T>(obj: T): ?T {
     return JSON.parse(JSON.stringify(obj));
 }
 
-let idSeq = -11111;
```

Comment:
all it needs is a unique id between nodes, i think this was because originally it was building a sample pipeline in code, this was to prevent overlap